Makefile

```diff
@@ -22,6 +22,12 @@
 # allow optional per-repo overrides
 -include Makefile.overrides.mk
 
+
+# Dynamic set sidcar
+# Now it support Envoy and MOSN, if set SIDCAR = MOSN, MOSN will be used as the proxy in Istio.
+export SIDCAR = MOSN
```

Comment:
SIDECAR -> sidecar
All the words were misspelled in the file.

Reply:
done.